cdk/package.json

```diff
@@ -15,15 +15,15 @@
     "@guardian/eslint-config-typescript": "^9.0.4",
     "@types/jest": "^29.5.12",
     "@types/node": "20.12.2",
-    "aws-cdk": "2.132.0",
-    "aws-cdk-lib": "2.132.0",
+    "aws-cdk": "2.134.0",
+    "aws-cdk-lib": "2.134.0",
     "constructs": "10.3.0",
     "eslint": "^8.57.0",
     "jest": "^29.7.0",
     "prettier": "^3.2.5",
     "source-map-support": "^0.5.21",
     "ts-jest": "^29.1.2",
     "ts-node": "^10.9.2",
-    "typescript": "5.4.3"
```

Comment:
I'm not entirely sure why we needed to downgrade to typescript 5.3.3, but @guardian/eslint-config-typescript started complaining about wanting typescript 5.3.3 in its peer deps.